packages/analitics/index.ts

```diff
@@ -0,0 +1,49 @@
+import { observe } from "frontity";
+import Analytics from "./types";
+
+type AfterCSR = Analytics["actions"]["analytics"]["afterCSR"];
+type SendPageview = Analytics["actions"]["analytics"]["sendPageview"];
+type SendEvent = Analytics["actions"]["analytics"]["sendEvent"];
+
+// Send a pageview anytime the link changes.
+export const afterCSR: AfterCSR = ({ state, actions }) => {
+  console.log("afterCSR");
+
+  observe(async () => {
+    const { link } = state.router;
+
+    // Wait data to be ready.
+    if (state.source.get(link).isReady) {
+      /**
+       * Execute action in the next tick so we can
+       * ensure the title has changed by React.
+       */
+      setTimeout(() => {
```

Comment:
I think this can be kind of dangerous... How can we make sure this is not going to break? What happens if React takes more time to update the `document.title`? Maybe we need to think of a better technique to extract the title.

Reply:
Here I (hastily) assumed that React would render syncronously as soon as `data.isReady` was `true`, so it would be `safe` to use `setTimeout` in this case.

Maybe I assumed wrong 😅